src/app/domain/models/deposit.py

```diff
@@ -0,0 +1,13 @@
+import uuid
+from django.db import models
+from django.utils import timezone
+
+from app.domain.models.account import Account
+
+
+class Deposit(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    account = models.ForeignKey(Account, on_delete=models.SET_NULL, null=True)
+    description = models.TextField(null=True)
```

Comment:
Nunca utilize um campo `CharField` ou `TextField` como `null=True`. É para ser utilizado com `blank=True`.

https://stackoverflow.com/questions/8609192/what-is-the-difference-between-null-true-and-blank-true-in-django

https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.Field.null

```suggestion
    description = models.TextField()
```